internal/namespaces/instance/v1/custom_server_test.go

```diff
@@ -223,14 +223,15 @@ func Test_ServerUpdateCustom(t *testing.T) {
 				assert.Equal(t, 0, len(ctx.Result.(*instance.UpdateServerResponse).Server.Volumes))
 			},
 			AfterFunc: core.AfterFuncCombine(
-				core.ExecAfterCmd(`scw instance delete volume volume-id={{ (index .Server.Volumes "0").ID }}`),
-				core.ExecAfterCmd(`scw instance delete volume volume-id={{ (index .Server.Volumes "1").ID }}`),
+				core.ExecAfterCmd(`scw instance volume delete volume-id={{ (index .Server.Volumes "0").ID }}`),
+				core.ExecAfterCmd(`scw instance volume delete volume-id={{ (index .Server.Volumes "1").ID }}`),
 				deleteServer("Server"),
 			),
 		}))
 	})
 }
 
+// these tests needs to be run in sequence
```

Comment:
Why ? 

Comments should explain `why`, not `what`.

Reply:
Good question :sweat_smile: it doesnt work in parallel but works in sequence. Not sure why :thinking: